src/main/js/helpers/region.js

```diff
@@ -246,14 +283,26 @@ const toggleRegion = (canvasSVG, key) =>
  * * If more than 1 target is displayed -> Hide regions
  * * If only 1 target is displayed -> show the region using unique data set key
  * @private
- * @param {Array} shownTargets - Targets/data sets that are currently displayed in graph
+ * @param {Object} graphConfig - Targets/data sets that are currently displayed in graph
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
  * @returns {Object} d3 svg path
  */
-const processRegions = (shownTargets, canvasSVG) =>
-    isSingleTargetDisplayed(shownTargets)
-        ? toggleRegion(canvasSVG, ...shownTargets)
+const processRegions = (graphConfig, canvasSVG) =>
+    isSingleTargetDisplayed(graphConfig.shownTargets)
+        ? toggleRegion(canvasSVG, ...graphConfig.shownTargets)
+        : checkRegion(graphConfig, canvasSVG);
```

Comment:
`checkAllRegions`.
Previously this line read like -> If single region then toggle this region or hide all regions. 
Now it will read like -> If single region then toggle or check all regions.

Reply:
ok i will rename it to **checkAllRegions**